_dashboards/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: About Dashboards
+title: Get Started with OpenSearch Dashboards
```

Comment:
Is the doc standard to say things like "Get Started" vs. "Getting Started"? This is rather unusual.

Reply:
Revised